### ISSUE TITLE
Set focus doesn't set keyboard focus in Windows 10

### ISSUE THREAD

**Issue by ScreamingHawk** [extr0py/node-process-windows#5]
I'm trying to send key commands to another application from within a nodeJS application. 

The application doesn't receive keyboard focus or come to the front. It only flashes in the task bar.

**Comment by ghost** [comment]
Same problem here :(, maybe am missing something... 
Hey Hawk, did you fix the problem? It is working for you?
D

**Comment by ScreamingHawk** [comment]
@dilbert007 Nope sorry. Decided to go with another solution entirely...

Looks like this project is dead. The open PR looks like it has a lot of fixes that might make this work https://github.com/adamsoutar/node-process-windows
